### DISCUSSION TITLE
Is there a way to get the same results as FuzzyWuzzy with WRatio

### DISCUSSION THREAD

**Discussion by Tom-Mosaic** [rapidfuzz/RapidFuzz#228] [Q&A] [answered]
Hey! I'm trying to replace fuzzywuzzy with rapidfuzz as it saves literal minutes of runtime on my code, but was hoping to get the same results. However, there are discrepancies of almost 10 points with some comparisons. Here's an example

```
caption = ['MUST NOT EXCEED 100 FEET (305 METRES}']
context = ['  :  Wiring Description  ,Drawing Number  ,Page','Prox Input  ,135243  ,22-3','"Prox Input, Ext Barriers  ",135245  ,22-5']

for cap in caption:
    fuzz_match = []
    for str_ in context:
        fuzz_match.append(((str_,round(rapidfuzz.fuzz.WRatio(cap, str_)))))

print(fuzz_match)
```
With the output being:
```
[('  :  Wiring Description  ,Drawing Number  ,Page', 31), ('Prox Input  ,135243  ,22-3', 33), ('"Prox Input, Ext Barriers  ",135245  ,22-5', 38)]
31
33
38
```
and fuzzywuzzy

```
fuz_match = extractWithoutOrder(caption[0], context) 
print(list(fuz_match))
```
with the output being: 
```
[('  :  Wiring Description  ,Drawing Number  ,Page', 23), ('Prox Input  ,135243  ,22-3', 27), ('"Prox Input, Ext Barriers  ",135245  ,22-5', 32)]
23
27
32
```

Some of the code matches, but a good half doesn't, is there something I could do to get it to match? 

NOTES:
1) I am a beginner coder and may have plain mistakes here, let me know what I could improve!
2) I know that rapidfuzz has a process.extract function as well, but with it outputting three variables made it really hard to implement the cade already existing, though if someone has a simple solution for that as well, that would be welcome! I've come close, but end up using almost the same amount of loops haha
3) I had run fuzzywuzzy in the same format with fuzz WRatio and two loops, and it came out as the same as the extractWithoutOrder function, so I know it doesn't matter which I use.

**Comment by maxbachmann** [comment, marked as answer]
fuzzywuzzy provides two different versions `fuzzywuzzy` and `fuzzywuzzy['speedup']`. If you install `fuzzywuzzy['speedup']`:
```
pip install fuzzywuzzy['speedup']
```
you will get the same results as with rapidfuzz

**Reply by Tom-Mosaic** [reply]
That did the trick! Thanks for getting back so fast
